testmain.py

```diff
@@ -0,0 +1,13 @@
+from zxy6005s import ZXY6005s
```

Comment:
Is this script intended for manual-regression testing?

If so, please move it and similar scripts to a `scripts/` subdir,